reportWebVitals.js

```diff
@@ -0,0 +1,15 @@
+const reportWebVitals = (onPerfEntry) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```

Comment:
- [ ] OPTIONAL:  This file is not required by the project.  I recommend you delete it.

Reply:
I did then it kept giving errors saying that it is missing  @Meltrust

Reply:
This is because the file is it's being imported and used on index.js which is totally unnecessary too! Look:

![image](https://user-images.githubusercontent.com/57421823/164149794-7e712bfc-9ae4-437c-b701-55d1a3b73bfa.png)

If you delete those 2 lines, you will be able to delete the file freely without errors.  

Of course, this is just a suggestion! Happy coding! 💪

Reply:
@Meltrust  thank you, that solution worked ...